bin/start

```diff
@@ -20,17 +20,19 @@ Usage: ./bin/start [options]
 
     --gke         Run the integration tests on GKE. By default the tests are run on GKE.
 
-    --oc311       Run the integration tests on Openshift v3.11. By default the tests are run on GKE.
+    --oldest     Run the integration tests on the oldest running Openshift platform. By default the tests are run on GKE.
```

Comment:
Silly nit: The spacing before the word `Run` is off by 1 for `--oldest` and `--next` options.
That's all I've got, everything else looks good to me (beyond comments already made).